openfermionpyscf/tests/test_run_pyscf.py

```diff
@@ -0,0 +1,45 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I think you should name it _test_run_pyscf.py not test_run_pyscf.py. Just for consistency with other files.